pkg/provider/aws/plugin/loadbalancer/auth.go

```diff
@@ -0,0 +1,27 @@
+package aws
```

Comment:
Do I still need this file?

Reply:
I'd leave this for now and we can clean it up to use a common pkg later...

Reply:
ok